app/topics/shared/treeview-node/treeview-node.component.ts

```diff
@@ -39,12 +37,16 @@ export class TreeView implements OnInit {
       );
   }
 
+  loadChildren()
```

Comment:
This function should do a server call to load the topics and not only hide the rest of the subtopics. 
So the getSubtopics method will need a pagination query as well.